tests/conftest.py

```diff
@@ -120,6 +124,10 @@ def _configure_bigchaindb(request):
         test_db_name = '{}_{}'.format(TEST_DB_NAME, xdist_suffix)
 
     backend = request.config.getoption('--database-backend')
+
+    if backend == 'mongodb-ssl':
+        backend = 'mongodb'
+
     config = {
```

Comment:
As discussed offline, removing all dependence on env variables.
We will still need `BIGCHAINDB_DATABASE_BACKEND` and `BIGCHAINDB_DATABASE_SSL`.
For ex:
```
BIGCHAINDB_DATABASE_BACKEND=mongodb \
BIGCHAINDB_DATABASE_SSL= \
pytest -sv --database-backend=mongodb -x --ff -vv
```
or 
```
BIGCHAINDB_DATABASE_BACKEND=mongodb \
BIGCHAINDB_DATABASE_SSL=true \
pytest -sv --database-backend=mongodb-ssl -x --ff -vv
```